### ISSUE TITLE
140 faces view : alt attribute fix

### ISSUE THREAD

**Issue by ushenoyuog** [ccswbs/hjckrrh#816]
The alt attribute is missing on the images in the 140 faces view, on all sites that use it. 
- SEDRD
- Business
- Grad Studies
- OAC

![image](https://user-images.githubusercontent.com/23462414/30277021-9658e9a0-96d4-11e7-9098-ef76824d6196.png)

Acceptance Criteria:
- Should have empty alt attribute. 
- Should not throw 'missing alt attribute ' issue on site improve

This **IS** a testable issue, but there's no point in writing a testcafe test since we're killing that framework. Will test manually.
